core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -12059,4 +12059,26 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
     final String expectedSparkQuery = "SELECT ~ (SAL) AS bit_not\nFROM scott.EMP";
     assertThat(toSql(root, DatabaseProduct.BIG_QUERY.getDialect()), isLinux(expectedSparkQuery));
   }
+  @Test public void testCountSetWithLiteralParameter() {
+    RelBuilder builder = relBuilder();
+    final RexNode rexNode = builder.call(SqlLibraryOperators.BIT_COUNT,
```

Comment:
can you please rename rexNode to something more meaningful

Reply:
done